connman/src/network.c

```diff
@@ -215,15 +215,17 @@ static void dhcp_failure(struct connman_network *network,
 		return;
 
 	ipconfig_ipv4 = __connman_service_get_ip4config(service);
+	if (!ipconfig_ipv4)
+		return;
+
 	if (ipconfig_ipv4 != ipconfig) {
 		DBG("ignore failure, ipconfig has changed");
 		return;
 	}
 
 	DBG("lease lost for ipconfig %p", ipconfig_ipv4);
 
-	if (!ipconfig_ipv4)
-		return;
+
```

Comment:
Adds extra empty line.

Reply:
Yep, missed that, thanks. Removed all extra empty lines and force-pushed.